### PR TITLE
chore: update our CMMN model XML file 

### PR DESCRIPTION
**Pull request by edgarvonk**
By importing and exporting it on Flowable Cloud. 

Solves PZ-976

### CODE REVIEW

File: src/main/resources/cmmn/Generiek_zaakafhandelmodel.cmmn.xml
Comment (bas-info-nl) [inline src/main/resources/cmmn/Generiek_zaakafhandelmodel.cmmn.xml:1]: I find it hard to do a good review on this, any hints what I can look for?

And is there a linter or formatter that we can use for XML files to ensure the structure of the files don't change depending on local configurations and tooling?

File: src/main/resources/cmmn/Generiek_zaakafhandelmodel.cmmn.xml [hunk 1]
```diff
@@ -1,191 +1,216 @@
...
+                <design:stencilid><![CDATA[ExpandedStage]]></design:stencilid>
+            </extensionElements>
+            <planItem id="planItemINTAKE" name="Intake" definitionRef="INTAKE">
                 <itemControl>
                     <requiredRule></requiredRule>
                 </itemControl>
             </planItem>
-            <planItem id="planItem11" name="In behandeling" definitionRef="IN_BEHANDELING">
-                <entryCriterion id="sid-563BE4E9-78ED-432A-9236-1D0708028133" sentryRef="sentry8"></entryCriterion>
+            <planItem id="planItemIN_BEHANDELING" name="In behandeling" definitionRef="IN_BEHANDELING">
```
Comment (bas-info-nl) [inline src/main/resources/cmmn/Generiek_zaakafhandelmodel.cmmn.xml:14]: different cases makes it harder to "read"

File: src/main/resources/cmmn/Generiek_zaakafhandelmodel.cmmn.xml [hunk 2]
```diff
@@ -1,191 +1,216 @@
...
-                <planItemOnPart id="sentryOnPart3" sourceRef="planItem4">
+            <sentry id="sentrysid-563BE4E9-78ED-432A-9236-1D0708028133">
+                <extensionElements>
+                    <design:stencilid><![CDATA[EntryCriterion]]></design:stencilid>
+                </extensionElements>
+                <planItemOnPart id="sentryOnPartsid-563BE4E9-78ED-432A-9236-1D0708028133" sourceRef="planItemINTAKE">
                     <standardEvent>complete</standardEvent>
                 </planItemOnPart>
-                <ifPart>
+                <ifPart id="sentryIfPart_sid-563BE4E9-78ED-432A-9236-1D0708028133">
```
Comment (bas-info-nl) [inline src/main/resources/cmmn/Generiek_zaakafhandelmodel.cmmn.xml:24]: Is this an id specifically added for Sentry? Why?

File: src/main/resources/cmmn/Generiek_zaakafhandelmodel.cmmn.xml [hunk 3]
```diff
@@ -1,191 +1,216 @@
...
             <stage id="INTAKE" name="Intake" autoComplete="true" flowable:displayOrder="1">
                 <extensionElements>
-                    <flowable:planItemLifecycleListener sourceState="available" targetState="active"
-                                                        class="net.atos.zac.flowable.cmmn.UpdateZaakLifecycleListener">
-                        <flowable:field name="status">
-                            <flowable:string><![CDATA[Intake]]></flowable:string>
-                        </flowable:field>
-                    </flowable:planItemLifecycleListener>
+                    <design:stencilid><![CDATA[ExpandedStage]]></design:stencilid>
+                    <flowable:planItemLifecycleListener sourceState="available" targetState="active" class="net.atos.zac.flowable.cmmn.UpdateZaakLifecycleListener"></flowable:planItemLifecycleListener>
```
Comment (bas-info-nl) [inline src/main/resources/cmmn/Generiek_zaakafhandelmodel.cmmn.xml:31]: Empty element: should this not be a single tag?

**Review by bas-info-nl — APPROVED** [review]

File: src/main/resources/cmmn/Generiek_zaakafhandelmodel.cmmn.xml (hunk 2 above)
Comment (edgarvonk) [reply src/main/resources/cmmn/Generiek_zaakafhandelmodel.cmmn.xml:24]: I have no idea. :-) I just copied this XML from the export of the Flowable Cloud editor. My assumption is that the Flowable Cloud editor is the only place where we will ever manage our CMMN model..

**Comment by edgarvonk** [comment]
Never mind. This CMMN model XML change broke ZAC.  I think it's the renaming of the task definitions in the XML file. ZAC relies on those. I will revert this PR.

**Pull request closed without merging**